api/authentication/models.py

```diff
@@ -43,7 +43,7 @@ def create_user(self, email, password=None, **extra_fields):
         return self._create_user(email, password, **extra_fields)
 
     def create_superuser(self, email, password, **extra_fields):
-        extra_fields.setdefault('is_staff', True)
+        extra_fields.setdefault('is_staff', False)
```

Comment:
`superuser` should have `is_staff True`

Reply:
I've set to True, we will create another field for moderator and function create_moderator()